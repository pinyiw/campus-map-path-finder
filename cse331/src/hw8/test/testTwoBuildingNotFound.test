r
HEY
HOW

r
ARE
YOU

q