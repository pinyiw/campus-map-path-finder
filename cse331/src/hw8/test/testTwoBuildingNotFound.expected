Menu:
r to find a route
b to see a list of all buildings
q to quit

Enter an option ('m' to see the menu): Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: HEY
Unknown building: HOW

Enter an option ('m' to see the menu):
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: ARE
Unknown building: YOU

Enter an option ('m' to see the menu):
